kubetest/main.go

```diff
@@ -495,7 +492,7 @@ func migrateGcpEnvAndOptions(o *options) error {
 			name:   "--gcp-network",
 		},
 		{
-			env:    nodeImage,
+			env:    "KUBE_NODE_OS_DISTRIBUTION",
 			option: &o.gcpNodeImage,
```

Comment:
What's the resolution order if o.gcpNodeImage is set *and* KUBE_NODE_OS_DISTRIBUTION is set?

I think some of the migration I did has both set now.